src/pixie_cli/pkg/utils/checks.go

```diff
@@ -258,6 +268,10 @@ var (
 			}
 		}
 
+		if clusterType == ClusterTypeOpenShift {
+			return errors.New("openshift cluster detected. Please note that a Security Context Constraint (SCC) is required to run Pixie. Install a SCC in the namespace designated for the Pixie install before continuing. See example on https://docs.px.dev/reference/admin/environment-configs/")
+		}
```

Comment:
shouldn't we allow the installation to go through after they install the SCC? won't this always block the install?

Reply:
Errors returned from these "cluster checks" causes the cli to print a warning and prompt the user to continue:

> Some cluster checks failed. Pixie may not work properly on your cluster. Continue with deploy? (y/n) [y] :